lib/stimulus_reflex/reflex.rb

```diff
@@ -32,8 +32,6 @@ def initialize(channel, reflex_data:)
     @headers = {}
 
     check_version!
-
-    params
```

Comment:
I think this line actually needs to stay in rather than being dropped, it's calling the `params` _method_ of this class (around line 145) which is (confusingly) doing something completely unrelated to the `params` property of the `reflex_data` object.

Reply:
Well, the method is still there and is called on L117. But what was the point of calling it inside `#initialize`?

I agree that generally speaking, it is confusing to see two sets of params referenced inside this class. However, that is how it also worked before my changes, and I did not dare refactoring that part due not having enough understanding of the code around the request. If you have ideas how to refactor it safely, please share your thoughts and I'll try to fix it.